install/k8s/install.sh

```diff
@@ -217,7 +217,7 @@ if [ "$aci_key" = "" ]; then
 	aci_key=./aci.key
 	echo "dummy" >$aci_key
 else
-	cp $aci_key ./aci.key
+	cp -u $aci_key ./aci.key
```

Comment:
This case wouldn't be hit unless someone attempts an installation (with ACI) which fails and retries it from the same extracted installer directory, but it doesn't hurt to add.